### PR TITLE
fix(docs): continue correcting nav buttons for caps MTA-1563

### PR DESCRIPTION
**Pull request by RoRoJ**
MTA-1536: 
Capitalization corrects for nav buttons in:
Storage, Network, IoT

### CODE REVIEW

File: iot/iot-hub/how-to/use-logging-kickstart.mdx [hunk 1]
```diff
@@ -98,6 +98,6 @@ The bridge also adds a `topic` and a `timestamp` field to any document.
 For more information about Kibana features, [see the official documentation](https://www.elastic.co/guide/en/kibana/current/index.html).
 
 <Navigation title="See Also">
-  <PreviousButton to="/iot/iot-hub/how-to/use-flow-programming-kickstart/">How to use the Flow-Programming Kickstart</PreviousButton>
-  <NextButton to="/iot/iot-hub/how-to/delete-route/">How to delete a Route</NextButton>
+  <PreviousButton to="/iot/iot-hub/how-to/use-flow-programming-kickstart/">How to use the flow-programming Kickstart</PreviousButton>
```
Comment (bene2k1) [inline iot/iot-hub/how-to/use-logging-kickstart.mdx:101]:
```suggestion
  <PreviousButton to="/iot/iot-hub/how-to/use-flow-programming-kickstart/">How to use the flow-programming kickstart</PreviousButton>
```

File: iot/iot-hub/how-to/use-flow-programming-kickstart.mdx [hunk 2]
```diff
@@ -88,6 +88,6 @@ To achieve this, an `MQTT input` "node" (with the correct settings) is connected
 Now you can create your own application to suit your needs. Check out the official [NodeRED documentation](https://nodered.org/docs/user-guide/) for more information.
 
 <Navigation title="See Also">
-  <PreviousButton to="/iot/iot-hub/how-to/use-faas-kickstart/">How to use the Function as-a-Service Kickstart</PreviousButton>
-  <NextButton to="/iot/iot-hub/how-to/use-logging-kickstart/">How to use the Logging Kickstart</NextButton>
+  <PreviousButton to="/iot/iot-hub/how-to/use-faas-kickstart/">How to use the function as-a-service Kickstart</PreviousButton>
+  <NextButton to="/iot/iot-hub/how-to/use-logging-kickstart/">How to use the logging Kickstart</NextButton>
```
Comment (bene2k1) [inline iot/iot-hub/how-to/use-flow-programming-kickstart.mdx:92]:
```suggestion
  <NextButton to="/iot/iot-hub/how-to/use-logging-kickstart/">How to use the logging kickstart</NextButton>
```

File: iot/iot-hub/how-to/use-faas-kickstart.mdx [hunk 3]
```diff
@@ -121,6 +121,6 @@ To create the trigger for your function follow these steps:
 3. Click **Deploy** to deploy your function
 
 <Navigation title="See Also">
-  <PreviousButton to="/iot/iot-hub/how-to/use-dataviz-kickstart/">How to use the Data Visualization Kickstart</PreviousButton>
-  <NextButton to="/iot/iot-hub/how-to/use-flow-programming-kickstart/">How to use the Flow-Programming Kickstart</NextButton>
+  <PreviousButton to="/iot/iot-hub/how-to/use-dataviz-kickstart/">How to use the data visualization Kickstart</PreviousButton>
+  <NextButton to="/iot/iot-hub/how-to/use-flow-programming-kickstart/">How to use the flow-programming Kickstart</NextButton>
```
Comment (bene2k1) [inline iot/iot-hub/how-to/use-faas-kickstart.mdx:125]:
```suggestion
  <NextButton to="/iot/iot-hub/how-to/use-flow-programming-kickstart/">How to use the flow-programming kickstart</NextButton>
```

File: iot/iot-hub/how-to/use-faas-kickstart.mdx [hunk 4]
```diff
@@ -121,6 +121,6 @@ To create the trigger for your function follow these steps:
 3. Click **Deploy** to deploy your function
 
 <Navigation title="See Also">
-  <PreviousButton to="/iot/iot-hub/how-to/use-dataviz-kickstart/">How to use the Data Visualization Kickstart</PreviousButton>
-  <NextButton to="/iot/iot-hub/how-to/use-flow-programming-kickstart/">How to use the Flow-Programming Kickstart</NextButton>
+  <PreviousButton to="/iot/iot-hub/how-to/use-dataviz-kickstart/">How to use the data visualization Kickstart</PreviousButton>
```
Comment (bene2k1) [inline iot/iot-hub/how-to/use-faas-kickstart.mdx:124]: kickstart?

**Review by bene2k1 — APPROVED** [review]

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**